arches_modular_reports/app/views/modular_report.py

```diff
@@ -42,12 +42,14 @@
 @method_decorator(can_read_resource_instance, name="dispatch")
 class ModularReportConfigView(View):
     def get(self, request):
-        """Just get first. But if there are multiple in the future,
-        the vue component will need to know which one to request."""
+        filters = Q(graph__resourceinstance=request.GET.get("resourceId"))
```

Comment:
We should probably add v8 compat while here with some sort of source_identifier=None check. Game to do that now?